back/engines/commercial/analytics/lib/query.rb

```diff
@@ -39,12 +40,13 @@ def model
 
     def all_dimensions
       @all_dimensions ||= model
-        .reflect_on_all_associations(:belongs_to)
+        .reflect_on_all_associations()
         .to_h do |assoc|
           [
             assoc.name.to_s,
             {
-              columns: "Analytics::#{assoc.options[:class_name]}".constantize.new.attributes.keys,
+              # Note: Changed this to remove 'Analytics::'
```

Comment:
In the context of the PR, I undestand what you mean here, but in a few months, I'm sure I'll be scratching my head  reading this comment. Wouldn't it be more appropriate as a commit message?